test/solidity/Facets/GlacisFacet.t.sol

```diff
@@ -0,0 +1,284 @@
+import { GlacisFacet } from "lifi/Facets/GlacisFacet.sol";
+import { IGlacisAirlift, QuoteSendInfo } from "lifi/Interfaces/IGlacisAirlift.sol";
+import { InsufficientBalance } from "lifi/Errors/GenericErrors.sol";
+
+// Stub GlacisFacet Contract
+contract TestGlacisFacet is GlacisFacet {
+    constructor(IGlacisAirlift _airlift) GlacisFacet(_airlift) {}
+
+    function addDex(address _dex) external {
+        LibAllowList.addAllowedContract(_dex);
+    }
+
+    function setFunctionApprovalBySignature(bytes4 _signature) external {
+        LibAllowList.addAllowedSelector(_signature);
+    }
+}
+
+contract GlacisFacetTest is TestBaseFacet {
+    GlacisFacet.GlacisData internal validGlacisData;
+    TestGlacisFacet internal glacisFacet;
+    ERC20 internal wormhole;
+    uint256 internal defaultWORMHOLEAmount;
+    uint256 internal tokenFee;
+
+    IGlacisAirlift internal constant airlift =
+        IGlacisAirlift(0xE0A049955E18CFfd09C826C2c2e965439B6Ab272);
+    address internal ADDRESS_WORMHOLE_TOKEN =
+        0xB0fFa8000886e57F86dd5264b9582b2Ad87b2b91;
+
+    uint256 internal payableAmount = 1 ether;
+
+    function setUp() public {
+        customRpcUrlForForking = "ETH_NODE_URI_ARBITRUM";
+        customBlockNumberForForking = 297418708;
+        initTestBase();
+
+        wormhole = ERC20(ADDRESS_WORMHOLE_TOKEN);
+
+        defaultWORMHOLEAmount = 1_000 * 10 ** wormhole.decimals();
+
+        deal(
+            ADDRESS_WORMHOLE_TOKEN,
+            USER_SENDER,
+            500_000 * 10 ** wormhole.decimals()
+        );
+
+        glacisFacet = new TestGlacisFacet(airlift);
+        bytes4[] memory functionSelectors = new bytes4[](4);
+        functionSelectors[0] = glacisFacet.startBridgeTokensViaGlacis.selector;
+        functionSelectors[1] = glacisFacet
+            .swapAndStartBridgeTokensViaGlacis
+            .selector;
+        functionSelectors[2] = glacisFacet.addDex.selector;
+        functionSelectors[3] = glacisFacet
+            .setFunctionApprovalBySignature
+            .selector;
+
+        addFacet(diamond, address(glacisFacet), functionSelectors);
+        glacisFacet = TestGlacisFacet(address(diamond));
+        glacisFacet.addDex(ADDRESS_UNISWAP);
+        glacisFacet.setFunctionApprovalBySignature(
+            uniswap.swapExactTokensForTokens.selector
+        );
+        glacisFacet.setFunctionApprovalBySignature(
+            uniswap.swapTokensForExactETH.selector
+        );
+        glacisFacet.setFunctionApprovalBySignature(
+            uniswap.swapETHForExactTokens.selector
+        );
+
+        setFacetAddressInTestBase(address(glacisFacet), "GlacisFacet");
+
+        // adjust bridgeData
+        bridgeData.bridge = "glacis";
+        bridgeData.sendingAssetId = ADDRESS_WORMHOLE_TOKEN;
+        bridgeData.minAmount = defaultWORMHOLEAmount;
+        bridgeData.destinationChainId = 10;
+
+        QuoteSendInfo memory quoteSendInfo = IGlacisAirlift(address(airlift))
```

Comment:
a comment would help here to understand why you make this call

Please try to comment extensively, they dont hurt but help to read files much faster